packages/shared/src/css/styles.css

```diff
@@ -1,10 +1,8 @@
 :root {
   --p-transition-duration: 0s;
-  --p-animation-duration__spinner: 0s;
-  --p-animation-duration__banner: 0s;
-  --p-override-popover-animation-duration: 0s;
-  --p-override-toast-skip-timeout: true;
-  --p-override-toast-animation-duration: 0s;
+  --p-animation-duration: 0s;
+  --p-temporary-spinner-stroke-dasharray: 40;
```

Comment:
Seems like this is not getting applied for button-pure vrt test?

Reply:
Could you specify? There is a partial dash array of the spinner in the button-pure vrt fixture or do i miss something?

Reply:
<img width="1180" alt="Screenshot 2023-01-25 at 16 29 22" src="https://user-images.githubusercontent.com/30120450/214604783-8b65957c-0545-4c9d-be5e-949d1f8efe9e.png">